homeassistant/components/light/limitlessled.py

```diff
@@ -0,0 +1,102 @@
+"""
+homeassistant.components.light.limitlessled
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Support for LimitlessLED bulbs, also known as...
+
+EasyBulb
+AppLight
+AppLamp
+MiLight
+LEDme
+dekolight
+iLight
+
+"""
+import random
+import logging
+
+from homeassistant.helpers.entity import ToggleEntity
+from homeassistant.const import STATE_ON, STATE_OFF, DEVICE_DEFAULT_NAME
+from homeassistant.components.light import ATTR_BRIGHTNESS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    try:
+        import ledcontroller
+    except ImportError:
+        _LOGGER.exception("Error while importing dependency ledcontroller.")
+        return
+
+    led = ledcontroller.LedController(config['host'])
```

Comment:
Is there a possibility to validate that the connection has been established successfully?